webprojects/snippets.md

```diff
@@ -0,0 +1,114 @@
+
+![Icons](/assets/images/webprojects/snippets/snippet_w_icon_examples.png)
+
+* Intangible: quality that is recognized but difficult to quantify.
+
+    * **Quikc**: get to the point with short words and sentences.
+    * **Clever**: find humor, irony and wordplay. Reference modern life.
+    * **Alert**: ready to offer tips, advice and guidance like a trained concierge.
+    * **Untamed**: be quirky. Do and say things no other brand would.
+
+In addition to the guidelines mentioned above in terms of localizing a snippet, it is very important that this is the result of a team collaborative effort. The proposed snippet should come in both target language and in English so the snippets team can understand what is being conveyed.
+
+Try to submit multiple variant snippets of the same message, or multiple snippets of different messages if you can, to lower the chances of having some snippets rejected.
+
+#### Can I use external icons and URLs in my snippet?
+
+You can pull icons from the [Firefox emoji bank](http://mozilla.github.io/fxemoji/dist/FirefoxEmoji/index.html) by specifying the icon number. This is the recommended list of icons to choose from.
+
+If you can’t find anything in the bank that fits, you can suggest icon ideas or even attach a custom icon that respects the guidelines. Suggesting new icons will increase review and approval time.
+
+Unless specified, the only non-Mozilla pages we can link to from snippets are Wikipedia pages.
+
+### File a request
+
+Once a snippet is formalized, you can [file a Snippets bug request](https://bugzilla.mozilla.org/enter_bug.cgi?assigned_to=nobody%40mozilla.org&bug_file_loc=http%3A%2F%2F&bug_ignored=0&bug_severity=normal&bug_status=NEW&cc=pmo%40mozilla.com&cc=francesco.lodolo%40gmail.com&cf_fx_iteration=---&cf_fx_points=---&cf_status_firefox55=---&cf_status_firefox56=---&cf_status_firefox57=---&cf_status_firefox_esr52=---&cf_tracking_firefox55=---&cf_tracking_firefox56=---&cf_tracking_firefox57=---&cf_tracking_firefox_esr52=---&cf_tracking_firefox_relnote=---&component=Service&contenttypemethod=autodetect&contenttypeselection=text%2Fplain&defined_groups=1&flag_type-4=X&flag_type-607=X&flag_type-800=X&flag_type-803=X&flag_type-864=X&flag_type-916=X&form_name=enter_bug&maketemplate=Remember%20values%20as%20bookmarkable%20template&op_sys=Unspecified&priority=--&product=Snippets&rep_platform=Unspecified&target_milestone=---&version=unspecified) for review and approval. Specify the language, provide copy of the message in target language and in English as reference and choice of icon, suggest preferred timeline to run the snippet. Make sure to include a link to use in the proposed snippet.
+
+### What is the lead time for a snippet request?
+
+The team needs a minimum of two weeks, if snippet needs to appear in target language. This assumes that the snippet doesn’t require much revision.
```

Comment:
Other than en-ZA, I also think there is a chance that we have English for India or even the Philippines that have their own content, not recycling en-US.